services/apps/template_consumer/src/main.ts

```diff
@@ -0,0 +1,34 @@
+import { Config } from '@crowd/standard'
+import { ServiceConsumer, Options } from '@crowd/consumer'
+
+const config: Config = {
+  producer: {
+    enabled: false,
+    idempotent: true,
+  },
+  temporal: {
+    enabled: true,
+  },
+  redis: {
+    enabled: false,
+  },
+}
+
+const options: Options = {
+  maxWaitTimeInMs: 1000,
+  retryPolicy: {
+    initialRetryTime: 1000,
+    maxRetryTime: 1000,
+    retries: 1000,
+  },
+}
+
+const svc = new ServiceConsumer(config, options)
+
+setImmediate(async () => {
+  await svc.start()
+
+  // await svc.consumer.run({
```

Comment:
is this required? why is this commented out?

Reply:
This is a template/example/demo service just to show it works. If not commented, linter and CI fails.

Reply:
so in a real consumer/worker this is required?